index.md

```diff
@@ -47,9 +47,9 @@ Our next session will take place via Zoom on: Wednesday 27th October, 13:00-15:0
 
 ### Helping out
 
-These sessions cannot be run without support from members of the community. If you're willing to spend a couple of hours every few weeks to join a session and help out by answering questions or offering technical advice to other members of the research community, you can [register as a helper via this form](https://forms.office.com/r/wuHDQWxef4).
+These sessions cannot be run without support from members of the community. If you're willing to spend a couple of hours every few weeks to join a session and help out by answering questions or offering technical advice to other members of the research community, you can [register as a helper via this form](https://forms.office.com/r/wuHDQWxef4) and sign up to help at a particular drop-in via [the wiki](https://github.com/RSdropin/RSdropinUK/wiki/Session-and-helper-schedule). We also have a rsdropin-helpers channel on the [ukrse Slack organisation](https://society-rse.org/join-us/) - come and say hi :wave:
```

Comment:
Just a couple of points on the edits to this paragraph:

 - I think I'd refer to the "UKRSE Slack workspace"
 - I also wonder if the link to sign up to Slack is clearer on the [contact page](https://society-rse.org/about/contact/) of the Society website since it's right at the top and I wonder if the other page might give the impression that you need to join the Society to sign up to Slack?
 - We could also add a link directly to the [`#rsdropin-helpers`](https://ukrse.slack.com/archives/C02JBEE3H36) channel, however, it's a private channel so we need to add new people - I guess this should also be mentioned.
 
I'm bundling the first two suggestions into the following suggested update but feel free to ignore this and edit as you think best:

```suggestion
These sessions cannot be run without support from members of the community. If you're willing to spend a couple of hours every few weeks to join a session and help out by answering questions or offering technical advice to other members of the research community, you can [register as a helper via this form](https://forms.office.com/r/wuHDQWxef4) and sign up to help at a particular drop-in via [the wiki](https://github.com/RSdropin/RSdropinUK/wiki/Session-and-helper-schedule). We also have a #rsdropin-helpers channel on the [UKRSE Slack workspace](https://society-rse.org/about/contact/) - come and say hi :wave: (we'll need to add you to the channel first since this is a private channel)
```